core/web/jobs_controller_test.go

```diff
@@ -801,6 +801,7 @@ func setupJobsControllerTests(t *testing.T) (ta *cltest.TestApplication, cc clte
 func setupEthClientForControllerTests(t *testing.T) *evmclimocks.Client {
 	ec := cltest.NewEthMocksWithStartupAssertions(t)
 	ec.On("PendingNonceAt", mock.Anything, mock.Anything).Return(uint64(0), nil).Maybe()
```

Comment:
Is this one still necessary?
```suggestion
```

Reply:
Yes! Needed in [SyncOnChain](https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/txmgr/nonce_tracker.go#L120)